src/start.js

```diff
@@ -79,16 +80,15 @@ let enableDebugger = process.argv.find(arg => arg.includes('enableDebugger'))
 const DEBUG_MODE = !!process.env.STETHOSCOPE_DEBUG
 
 const focusOrCreateWindow = (mainWindow) => {
-  if (mainWindow && !mainWindow.isDestroyed()) {
+  if (mainWindow) {
     if (mainWindow.isMinimized()) {
-      mainWindow.restore()
+      return mainWindow.restore()
```

Comment:
It looks like the restore method doesn't have a return value. We should return `mainWindow` so the uses of this method will keep their local state correctly. I believe this will cause issues if you close the window and select check for updated by right-clicking the dock icon.

Reply:
Good catch!!